tools/storm/scripts/capture_screenshot/capture_screenshot.go

```diff
@@ -20,44 +17,9 @@ type CaptureScreenshotScript struct {
 }
 
 func (s *CaptureScreenshotScript) Run() error {
-	ppmFilename, err := os.CreateTemp("", "ppm")
+	err := stormutils.CaptureScreenshot(s.VmName, s.ArtifactsFolder, s.ScreenshotFilename)
 	if err != nil {
 		return fmt.Errorf("failed to create temporary file: %w", err)
 	}
-	ppmFilename.Close()
-	defer os.Remove(ppmFilename.Name())
-
-	err = capturePpmScreenshot(s.VmName, ppmFilename.Name())
-	if err != nil {
-		return err
-	}
-
-	err = os.MkdirAll(s.ArtifactsFolder, 0755)
-	if err != nil {
-		return err
-	}
-
-	pngPath := filepath.Join(s.ArtifactsFolder, s.ScreenshotFilename)
-	if err := convertPpmToPng(ppmFilename.Name(), pngPath); err != nil {
-		return err
-	}
-	return nil
-}
-
-func capturePpmScreenshot(vmName string, ppmFilename string) error {
-	virshOutput, virshErr := exec.Command("sudo", "virsh", "screenshot", vmName, ppmFilename).CombinedOutput()
```

Comment:
>virshOutput [](http://example.com/codeflow?start=1&length=11)

Would it make sense to put some info into the description as to what is wrong with the current method?